CONTRIBUTING.md

```diff
@@ -1,102 +1,77 @@
 We're glad you are interested in contributing to this project. We hope that this
 document helps you get started.
 
-If something is missing, incorrect, or made irrelevant please feel free to make
-a PR to keep it up-to-date.
+## Project Policies
 
-## Prerequisites
+This repository adheres to the following project policies:
 
-- [Go](https://golang.org/dl/)
-    - including [Git](https://git-scm.com/)
-- [Docker](https://www.docker.com/)
+- [Code of Conduct](https://github.com/buildpacks/.github/blob/master/CODE_OF_CONDUCT.md)
+- [Security](https://github.com/buildpacks/.github/blob/master/SECURITY.md)
+- [Support](https://github.com/buildpacks/.github/blob/master/SUPPORT.md)
 
-## Development
+## Pull Request Process
 
-### Building
+1. [Fork][fork] the repository.
+2. Create a branch for the issue: `git checkout -b {{BRANCH_NAME}}`
+3. Make any changes deemed necessary.
+4. Commit your changes: `git commit -s`\
+    _Learn more about the [sign-off](#sign-off-process) process below._
+5. Push to GitHub: `git push origin {{BRANCH_NAME}}`
+6. [Create the pull request][create-pr].
```

Comment:
Should there be something here that links to the development section?

Reply:
Also we used to have some guidance around preferring PRs to have a single commit. Are we moving away from that?

Reply:
> Also we used to have some guidance around preferring PRs to have a single commit. Are we moving away from that?

We are moving away from that. It ended up being to heavy handed. Some contributors prefer to "show their work".

Reply:
> Should there be something here that links to the development section?

Good idea. I made a few changes. Want to look at it again?